packages/sheets/src/services/permission/worksheet-permission/protect-background-img.ts

```diff
@@ -0,0 +1,17 @@
nodd%22%20clip-rule%3D%22evenodd%22%20d%3D%22M128.544%20654.332L552.808%20230.068L553.515%20230.775L129.251%20655.039L128.544%20654.332Z%22%20fill%3D%22%23ECECEC%22%2F%3E%0A%3Cpath%20fill-rule%3D%22evenodd%22%20clip-rule%3D%22evenodd%22%20d%3D%22M130.665%20656.454L554.929%20232.189L555.637%20232.897L131.372%20657.161L130.665%20656.454Z%22%20fill%3D%22%23ECECEC%22%2F%3E%0A%3Cpath%20fill-rule%3D%22evenodd%22%20clip-rule%3D%22evenodd%22%20d%3D%22M132.787%20658.575L557.051%20234.311L557.758%20235.018L133.494%20659.282L132.787%20658.575Z%22%20fill%3D%22%23ECECEC%22%2F%3E%0A%3Cpath%20fill-rule%3D%22evenodd%22%20clip-rule%3D%22evenodd%22%20d%3D%22M134.908%20660.696L559.172%20236.432L559.879%20237.139L135.615%20661.403L134.908%20660.696Z%22%20fill%3D%22%23ECECEC%22%2F%3E%0A%3Cpath%20fill-rule%3D%22evenodd%22%20clip-rule%3D%22evenodd%22%20d%3D%22M137.029%20662.817L561.293%20238.553L562.001%20239.261L137.736%20663.525L137.029%20662.817Z%22%20fill%3D%22%23ECECEC%22%2F%3E%0A%3Cpath%20fill-rule%3D%22evenodd%22%20clip-rule%3D%22evenodd%22%20d%3D%22M139.151%20664.939L563.415%20240.675L564.122%20241.382L139.858%20665.646L139.151%20664.939Z%22%20fill%3D%22%23ECECEC%22%2F%3E%0A%3Cpath%20fill-rule%3D%22evenodd%22%20clip-rule%3D%22evenodd%22%20d%3D%22M141.272%20667.06L565.536%20242.796L566.243%20243.503L141.979%20667.767L141.272%20667.06Z%22%20fill%3D%22%23ECECEC%22%2F%3E%0A%3Cpath%20fill-rule%3D%22evenodd%22%20clip-rule%3D%22evenodd%22%20d%3D%22M143.393%20669.181L567.657%20244.917L568.365%20245.624L144.101%20669.889L143.393%20669.181Z%22%20fill%3D%22%23ECECEC%22%2F%3E%0A%3Cpath%20fill-rule%3D%22evenodd%22%20clip-rule%3D%22evenodd%22%20d%3D%22M145.515%20671.303L569.779%20247.039L570.486%20247.746L146.222%20672.01L145.515%20671.303Z%22%20fill%3D%22%23ECECEC%22%2F%3E%0A%3Cpath%20fill-rule%3D%22evenodd%22%20clip-rule%3D%22evenodd%22%20d%3D%22M147.636%20673.424L571.9%20249.16L572.607%20249.867L148.343%20674.131L147.636%20673.424Z%22%20fill%3D%22%23ECECEC%22%2F%3E%0A%3Cpath%20fill-rule%3D%22evenodd%22%20clip-rule%3D%22evenodd%22%20d%3D%22M149.757%20675.545L574.021%20251.281L574.729%20251.988L150.465%20676.252L149.757%20675.545Z%22%20fill%3D%22%23ECECEC%22%2F%3E%0A%3Cpath%20fill-rule%3D%22evenodd%22%20clip-rule%3D%22evenodd%22%20d%3D%22M151.879%20677.667L576.143%20253.403L576.85%20254.11L152.586%20678.374L151.879%20677.667Z%22%20fill%3D%22%23ECECEC%22%2F%3E%0A%3Cpath%20fill-rule%3D%22evenodd%22%20clip-rule%3D%22evenodd%22%20d%3D%22M154%20679.788L578.264%20255.524L578.971%20256.231L154.707%20680.495L154%20679.788Z%22%20fill%3D%22%23ECECEC%22%2F%3E%0A%3Cpath%20fill-rule%3D%22evenodd%22%20clip-rule%3D%22evenodd%22%20d%3D%22M156.121%20681.909L580.385%20257.645L581.093%20258.352L156.829%20682.616L156.121%20681.909Z%22%20fill%3D%22%23ECECEC%22%2F%3E%0A%3Cpath%20fill-rule%3D%22evenodd%22%20clip-rule%3D%22evenodd%22%20d%3D%22M158.243%20684.031L582.507%20259.767L583.214%20260.474L158.95%20684.738L158.243%20684.031Z%22%20fill%3D%22%23ECECEC%22%2F%3E%0A%3C%2Fg%3E%0A%3C%2Fsvg%3E%0A';
```

Comment:
Can we use some scripts to auto generate this? Like those for conditional formatting.